es-bs-base/scss/_variables.scss

```diff
@@ -528,7 +526,7 @@ $h6-font-size-desktop:        $font-size-base * 0.875;
 
 $headings-margin-bottom:      1rem !default;
 $headings-font-family:        $font-family-sans-serif !default;
-$headings-font-weight:        700 !default;
+$headings-font-weight:        $font-weight-semibold !default;
```

Comment:
I think they want the 600 weight of `$font-weight-bold` rather than the 500 weight of `$font-weight-semibold`.

Reply:
I thought so too but I clarified; https://energysage.atlassian.net/browse/CED-428